map2loop/mapdata.py

```diff
@@ -495,29 +497,36 @@ def __retrieve_tif(self, filename: str):
             _type_: The open geotiff in a gdal handler
         """
         self.__check_and_create_tmp_path()
+        
         # For gdal debugging use exceptions
         gdal.UseExceptions()
         bb_ll = tuple(self.bounding_box_polygon.to_crs("EPSG:4326").geometry.total_bounds)
-        # try:
+    
         if filename.lower() == "aus" or filename.lower() == "au":
-            url = "http://services.ga.gov.au/gis/services/DEM_SRTM_1Second_over_Bathymetry_Topography/MapServer/WCSServer?"
+
+            url = "http://services.ga.gov.au/gis/services/DEM_SRTM_1Second/MapServer/WCSServer?"
```

Comment:
why is the url hardcoded in the mapdata module? Would it be better to set this in the config file?

Reply:
Not sure why, as this was originally written by @RoyThomsonMonash. 

@lachlangrose would the advantage of having it in the config file be that the user could change the url if the GA server is not available?

Reply:
This is for when you specify the DTM to be "aus" or "au" so it is hardcoded to the known GA server.  Having it is a user made config file seems unnecessary as they can already specify their own DTM file or url to use.  If the server changes more regularly then we could put it in a separate provided config file (or preferably the url to the aus_state_urls.py file though it doesn't really relate to individual states).

The choice to use the bathymetry version is so that if a map is semi over the ocean then it doesn't default the altitude to sealevel (0), I thought this preferable when dealing with geology, is that incorrect?

Reply:
@RoyThomsonMonash I was wrong, I meant put it in the aus_state_url file. You are right it shouldn't be a config parameter.

@AngRodrigues moving it to a file with the other URLs will makes updating the code in the future easier.

Reply:
@lachlangrose that is a good idea. Will add that to the backlog.